doc/modules/ensemble.rst

```diff
@@ -1461,22 +1463,29 @@ computationally expensive.
    Multiple stacking layers can be achieved by assigning `final_estimator` to
    a :class:`StackingClassifier` or :class:`StackingRegressor`::
 
+    >>> final_layer_rfr = RandomForestRegressor(
+    ...                        n_estimators=10, max_features=1,
+    ...                        max_leaf_nodes=5,random_state=42)
+    >>> final_layer_gbr = GradientBoostingRegressor(
+    ...                        n_estimators=10, max_features=1,
+    ...                        max_leaf_nodes=5,random_state=42)
```

Comment:
```suggestion
    >>> final_layer_rfr = RandomForestRegressor(
    ...     n_estimators=10, max_features=1, max_leaf_nodes=5,
    ...     random_state=42)
    >>> final_layer_gbr = GradientBoostingRegressor(
    ...     n_estimators=10, max_features=1, max_leaf_nodes=5,
    ...     random_state=42)
```

Reply:
Thank you! I'm always unsure of formatting in these situations, this was helpful.